frontend/packages/kubevirt-plugin/integration-tests/v2v/v2v.configs.ts

```diff
@@ -18,3 +18,20 @@ export const vmwareVMConfig: VMImportConfig = {
   operatingSystem: OperatingSystem.RHEL7_6,
   workloadProfile: WorkloadProfile.DESKTOP,
 };
+
+export const vmwareVMMultiNicConfig: VMImportConfig = {
+  name: 'v2v-rhel7-2nic-2disk-igor-imported',
+  sourceVMName: 'v2v-rhel7-2nic-2disk-igor',
+  provider: 'VMware',
+  instanceConfig: {
+    instance: IMPORT_WIZARD_CONN_TO_NEW_INSTANCE,
+    hostname: V2V_INSTANCE_HOSTNAME,
+    username: V2V_INSTANCE_USERNAME,
+    password: V2V_INSTANCE_PASSWORD,
+    saveInstance: true,
+  },
+  operatingSystem: OperatingSystem.RHEL7_6,
+  workloadProfile: WorkloadProfile.SERVER,
+};
+
+export const importConfig = [vmwareVMConfig, vmwareVMMultiNicConfig];
```

Comment:
`v2vImportConfigs` may be more suitable name.

Reply:
Fixed